### ISSUE TITLE
Create email addresses @nlpsandbox.io

### ISSUE THREAD

**Issue by tschaffter** [nlpsandbox/nlpsandbox#32]
Example:

- admin@nlpsandbox.io
- team@nlpsandbox.io

I'm currently using admin@nlpsandbox.io on the website pages for prospective partner organizations.

**Comment by tschaffter** [comment]
A suggestion from AWS WorkMail doc:

> As a best practice, you should add aliases for postmaster@ and abuse@. You can create distribution groups for these aliases if you want certain users in your organization to receive mail sent to these aliases.

**Comment by tschaffter** [comment]
I created an AWS WorkMail organization and three user accounts ($4/user/month):

- postmaster@nlpsandbox.io
- abuse@nlpsandbox.io
- team@nlpsandbox.io

The credentials to these account have been added to Sage favorite password manager.

I will look into configuring redirection to my @sagebase.org email address (I'll add a note to the credentials in the password manager).

> RFC 5321 states that errors during the processing of email should be directed to the postmaster email address. Likewise, RFC 2142 indicates that any abusive behaviors, such as spamming, should be reported to the abuse email address. 

[Source](https://help.returnpath.com/hc/en-us/articles/220223268-As-a-certified-sender-why-do-I-need-abuse-and-postmaster-role-accounts-)

We could add the following aliases in the future:

- security@nlpsandbox.io and add it to [SECURITY.md in GitHub repositories](https://docs.github.com/en/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository).
